mlir/lib/Dialect/Linalg/IR/LinalgOps.cpp

```diff
@@ -149,15 +154,41 @@ static void fillStructuredOpRegion(OpBuilder &opBuilder, Region &region,
   // iterator_types is an auto-generated method.
 }
 
+/// Helper to create a typical indexing map for MatmulOp. Returns a list of
+/// AffineMap.
+static SmallVector<AffineMap, 3>
+inferDefaultIndexingMaps(MLIRContext *context) {
```

Comment:
Nit: Rename this to `getDefaultIndexingMapsForMatmul`.

Reply:
Done.